ignite/__init__.py

```diff
@@ -6,4 +6,4 @@
 import ignite.metrics
 import ignite.utils
 
-__version__ = "0.5.0"
+__version__ = "0.4.6"
```

Comment:
Let's revert this change, as it is master's version

Reply:
Made the change. Please review